mne/viz/topo.py

```diff
@@ -853,23 +857,44 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
     -------
     fig : instance of matplotlib.figure.Figure
```

Comment:
I know our policy says "avoid unnecessary cosmetic changes in PRs", but since you're already changing this docstring, could you add a cross-reference to `mne.Epochs` and `matplotlib.figure.Figure` here?

Reply:
OK, I also will add one for `matplotlib.pyplot.imshow` OK?